src/pages/lists/action-list/action-list-local-actions/scorecard.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import * as Colors from '@brightlayer-ui/colors';
+import { Hero, HeroBanner, ScoreCard } from '@brightlayer-ui/react-components';
+import { GradeA, Temp, Moisture as Humidity } from '@brightlayer-ui/icons-mui';
+import Button from '@material-ui/core/Button';
+import { createStyles, makeStyles, Theme, useTheme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        scorecardRoot: {
+            width: 350,
+            height: 250,
+            marginTop: 16,
+            marginLeft: 16,
+        },
+        scorecardHeader: {
+            display: 'flex',
+            height: 48,
+            borderBottom: `1px solid ${theme.palette.divider}`,
+        },
+        scorecardHeaderContent: {
+            padding: '0 16px',
+            alignItems: 'center',
+        },
+        bodyWrapper: {
+            flex: 0,
+        },
+        badgeWrapper: {
+            flex: 1,
+        },
+    })
+);
+
+export const LocalActionsScoreCard = (): JSX.Element => {
+    const theme = useTheme();
+    const classes = useStyles(theme);
+
+    return (
+        <ScoreCard
```

Comment:
Must say I am really surprised to see you using scorecard for this... but looks like it gets the job done. Just a comment.